apis/fluent/atrium-api-fluent/src/jvmMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/sqlExceptionFeatureExtractors.kt

```diff
@@ -0,0 +1,41 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.creating.FeatureExpect
+import java.sql.SQLException
+
+/**
+ * Extracts the [errorCode][SQLException.getErrorCode] property from the subject of the assertion.
```

Comment:
Please copy the kdoc from collectionFeatureExtractors. I prefer if they are all the same, this way changes in the future can be made via search and replace